### PR TITLE
Base64

### PR DESCRIPTION
**Pull request by hemanthkumar650**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,9 @@
 {
   "dependencies": {
-    "express": "^4.18.2"
+    "base64-url": "^2.3.3",
+    "base64url": "^3.0.1",
```
Comment (0x-pranay) [inline package.json]: Remove these unused dependencies.

Comment (hemanthkumar650) [reply package.json]: Updated

File: package-lock.json [hunk 2]
```diff
@@ -29,6 +30,14 @@
...
       "integrity": "sha512-PCVAQswWemu6UdxsDFFX/+gVeYqKAod3D3UVm91jHwynguOwAvYPhx8nNlM++NqRcK6CxxpUafjmhIdKiHibqg=="
     },
+    "node_modules/base64url": {
+      "version": "3.0.1",
+      "resolved": "https://registry.npmjs.org/base64url/-/base64url-3.0.1.tgz",
+      "integrity": "sha512-ir1UPr3dkwexU7FdV8qBBbNDRUhMmIekYMFZfi+C/sLNnRESKPl23nB9b2pltqfOQNnGzsDdId90AEtG5tCx4A==",
+      "engines": {
+        "node": ">=6.0.0"
+      }
+    },
```
Comment (0x-pranay) [inline package-lock.json]: the package is still present in package-lock.json.

File: package.json [hunk 3]
```diff
@@ -1,6 +1,7 @@
 {
   "dependencies": {
-    "express": "^4.18.2"
+    "express": "^4.18.2",
+    "node.js": "^0.0.1-security"
```
Comment (0x-pranay) [inline package.json]: Why do we need  this `node.js` package ?
If its not used anywhere then we need to remove it

**Pull request merged**